book/4 - MultiWorlds/3 - Linking Worlds with a Plugin.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The easiest way to link worlds together is using a dedicated plugin, such as <a href="http://forum.mc-server.org/showthread.php?tid=2157">Portal V2</a>. More plugins can be found at the <a href="http://forum.mc-server.org/forumdisplay.php?fid=2">respository</a>.
```

Comment:
> Portal V2

Is V2 part of the name or the version? If it's the name that's ok, if it's the version, then we should say `Portal by $creator`.